src/utils/transactionUtils/transactionUtils.tsx

```diff
@@ -694,3 +695,34 @@ export function userHasSufficientBalance<T extends Record<string, number>>({
 export function mioToIo(mio: number): number {
   return mio / 1_000_000;
 }
+
+export function createDefaultAntState(state: any) {
+  return {
+    ticker: 'aos',
+    name: 'ANT',
+    controllers: [],
+    balances: {},
+    owner: '',
+    records: {
+      ['@']: {
+        transactionId: LANDING_PAGE_TXID,
+        ttlSeconds: 3600,
+      },
+    },
+    ...state,
+  };
+}
+
+export function createAntStateForOwner(owner: string, targetId: string) {
+  return createDefaultAntState({
+    owner: owner.toString(),
```

Comment:
toString() not necessary here in this function when it's already a string argument